core/controllers/feedback.py

```diff
@@ -70,7 +76,7 @@ def post(self, exploration_id, thread_id):
                 'Text for the message must be specified.')
 
         feedback_services.create_message(
-            thread_id,
+            '.'.join((exploration_id, thread_id)),
```

Comment:
Two things: could create_message() take two args instead (an exp id and a thread id), and, if the full thread id needs to be constructed by create_message(), can it be done in feedback_services similar to how create_thread() does it (i.e. by calling FeedbackThreadModel.generate_new_thread_id())?